### PR TITLE
Update Discord Nav/Footer Link

### PR DESCRIPTION
**Pull request by mmcbride2929**
# TL/DR
This PR updates the discord link within the Nav & Footer to navigate the user to the FCC discord invite page

# Overview of Change

`links.js`
- update discord href to navigate to discord invite screen

`Header.test.js`
- imported screen & render from testing-library
- wrote test to make sure we are rendering 'Discord' as the third item

`Footer.test.js`
- imported screen & render from testing-library
- wrote test to make sure we are rendering 'Discord' as the third item

### CODE REVIEW

File: src/links.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ export default [
...
   { href: '#about', text: 'About' },
   { href: 'https://www.freecodecamp.org/', text: 'FreeCodeCamp' },
-  { href: 'https://discord.gg', text: 'Discord' },
+  { href: 'https://discord.gg/aF8skEe9nq', text: 'Discord' },
   { href: 'https://github.com/FCCColumbus', text: 'GitHub' },
   { href: '#events', text: 'Events' },
-  { href: 'https://github.com/FCCColumbus/cbus-web/graphs/contributors', text: 'Contributors' },
-]
\ No newline at end of file
+  {href: 'https://github.com/FCCColumbus/cbus-web/graphs/contributors',text:'Contributors' },
```
Comment (jduffey) [inline src/links.js:8]: Minor nit: fix spacing between open brace and `href`, and space between comma and `text`

File: src/links.js [hunk 2]
```diff
@@ -2,8 +2,8 @@ export default [
   { href: '/', text: 'Home' },
   { href: '#about', text: 'About' },
   { href: 'https://www.freecodecamp.org/', text: 'FreeCodeCamp' },
-  { href: 'https://discord.gg', text: 'Discord' },
+  { href: 'https://discord.gg/aF8skEe9nq', text: 'Discord' },
```
[the remark this answers is not in the document]
Comment (mmcbride2929) [reply src/links.js:5]: Good call, I didn't even know it would link back to my account. I did set it to never expire, however I agree we should have the invited come from the admin account.

File: src/components/Events.jsx
Comment (readysetagile) [inline src/components/Events.jsx:1]: What is the purpose of these changes?

File: src/links.js [hunk 3]
```diff
@@ -2,8 +2,11 @@ export default [
...
   { href: 'https://github.com/FCCColumbus', text: 'GitHub' },
   { href: '#events', text: 'Events' },
-  { href: 'https://github.com/FCCColumbus/cbus-web/graphs/contributors', text: 'Contributors' },
-]
\ No newline at end of file
+  {
+    href: 'https://github.com/FCCColumbus/cbus-web/graphs/contributors',
```
Comment (readysetagile) [inline src/links.js]: what is the purpose of this change?

File: src/test/__snapshots__/App.test.js.snap
Comment (readysetagile) [inline src/test/__snapshots__/App.test.js.snap:1]: please use the correct link from the requirement in #50

File: src/test/components/Footer.test.js [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
+  it('renders', () => {
     const view = renderer.create(<Footer />);
     expect(view).toMatchSnapshot();
   });
+
+  it('displays "Discord" as the third item', () => {
+    render(<Footer />);
+    const thirdItem = screen.getByText('Discord');
+    expect(thirdItem).toBeInTheDocument();
+  });
```
Comment (readysetagile) [inline src/test/components/Footer.test.js:15]: this test is not needed since the "third link" test is covered in the snapshot

File: src/test/components/__snapshots__/Footer.test.js.snap [hunk 5]
```diff
@@ -20,7 +20,7 @@ exports[`Footer renders 1`] = `
       </li>
       <li>
         <a
-          href="https://discord.gg"
+          href="https://discord.gg/aF8skEe9nq"
```
Comment (readysetagile) [inline src/test/components/__snapshots__/Footer.test.js.snap:23]: please use the correct link from the requirement in #50

**Review by readysetagile — CHANGES_REQUESTED** [review]
comments within.  
- use the correct link
- remove the "third link" test
- remove the changed files which did not require changes

File: src/components/Events.jsx
Comment (readysetagile) [reply src/components/Events.jsx:1]: oh ic -- just remove the file from the commit

File: src/links.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jduffey) [reply src/links.js]: Screenshot 2023-08-02 at 11 48 43 AM

Uploading a photo to verify that the new Discord link is one generated by John. Resolving convo.

**Review by jduffey — APPROVED** [review]
All comments were addressed. Discord link now points to John's invite link. Approving.

**Review by readysetagile — APPROVED** [review]
